app/Commands/Component/UpdateComponentCommand.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/*
+ * This file is part of Cachet.
+ *
+ * (c) Alt Three Services Limited
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace CachetHQ\Cachet\Commands\Component;
+
+use CachetHQ\Cachet\Models\Component;
+
+class UpdateComponentCommand
+{
+    /**
+     * The component.
+     *
+     * @var \CachetHQ\Cachet\Models\Component
+     */
+    public $component;
+
+    /**
+     * The component name.
+     *
+     * @var string
+     */
+    public $name;
+
+    /**
+     * The component description.
+     *
+     * @var string
+     */
+    public $description;
+
+    /**
+     * The component status.
+     *
+     * @var int
+     */
+    public $status;
+
+    /**
+     * The component link.
+     *
+     * @var string
+     */
+    public $link;
+
+    /**
+     * The component order.
+     *
+     * @var int
+     */
+    public $order;
+
+    /**
+     * The component group.
+     *
+     * @var int
+     */
+    public $group_id;
+
+    /**
+     * The validation rules.
+     *
+     * @var string[]
+     */
+    public $rules = [
+        'name'   => 'string',
+        'status' => 'integer',
+        'link'   => 'url',
```

Comment:
you should include every property here apart from the objects, so including "order" and "group_id"